api/desecapi/management/commands/stop-abuse.py

```diff
@@ -0,0 +1,21 @@
+from django.core.management import BaseCommand
+
+from desecapi.models import RRset, Domain, User
+from desecapi.pdns_change_tracker import PDNSChangeTracker
+
+
+class Command(BaseCommand):
+    help = 'Removes all DNS records from the given domain and suspends the associated user'
+
+    def add_arguments(self, parser):
+        parser.add_argument('domain-name', nargs='*', help='Domain(s) to remove all DNS records from')
+
+    def handle(self, *args, **options):
+        with PDNSChangeTracker():
+            domain_f = Domain.objects.filter(name__in=options['domain-name'])
+            user_f = User.objects.filter(domains__name__in=options['domain-name'])
+            rrset_f = RRset.objects.filter(domain__name__in=options['domain-name']).exclude(type='NS', subname='')
```

Comment:
`domains__in=domain_f` (and `domain__in=domain_f`)?

Also, I find `_f` confusing. It looks like a file handle. But can stay that way :) No need to spend time on.

Reply:
done